config_rabbit.yml

```diff
@@ -19,7 +21,7 @@ Rabbit:
   ConnectionRetry:
     ElapsedTime: 2
     Interval: 20
-Mode: receiver
+Mode: sender
```

Comment:
Can it be in both modes simultaneously? Or it will be 2 different instances?

Reply:
for now -  different instances, may consider having it in both in future